package.json

```diff
@@ -142,6 +142,7 @@
     "elliptic": "^6.4.1",
     "form-data": "^2.3.3",
     "jsontokens": "^1.0.0",
+    "lodash": "^4.17.11",
```

Comment:
Looks like an accidental auto-import typo here

Reply:
See above comments on the linter.